content/_data/changelogs/lts.yml

```diff
@@ -4389,6 +4389,317 @@
 
+- version: "2.263.1"
+  date: 2020-12-03
+  lts_predecessor: "2.249.3"
+  lts_baseline: "2.263"
+
+  changes: # compared to lts_baseline 2.263 - extracted from the RC commit(s)
+
+  - type: major bug
+    category: major bug
+    pull: 5046
+    issue: 64040
+    authors:
+    - timja
+    references:
+      - pull: 5046
+      - issue: 64040
+      - issue: 64136
+    message: |-
+      Fix hidden page elements from radio blocks showing up when they should not.
+  - type: bug
+    category: bug
+    pull: 5038
+    issue: 62985
+    authors:
+    - jglick
+    references:
+      - pull: 5038
+      - url: https://github.com/stapler/stapler/releases/tag/stapler-parent-1.261
+        title: Stapler 1.261 changelog
+    message: |-
+      Fix file handle leak when viewing corrupted build logs.
+      Upgrade Stapler from 1.260 to 1.261.
+  - type: bug
+    category: bug
+    pull: 5013
+    issue: 63868
+    authors:
+    - oleg-nenashev
+    message: |-
+      Prevent the Build History Widget from crashing when users have Discover permissions without Read for folders.
+  - type: rfe
+    category: rfe
+    pull: 5034
+    issue: 64035
+    authors:
+    - oleg-nenashev
+    - olamy
+    references:
+      - issue: 64035
+      - pull: 5034
+      - pull: 4975
+      - url: https://github.com/jenkinsci/winstone/releases/tag/winstone-5.12
+        title: Winstone 5.12 changelog
+      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.31.v20200723
+        title: Jetty 9.4.31.v20200723 changelog
+      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.32.v20200930
+        title: Jetty 9.4.32.v20200930 changelog
+      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.33.v20201020
+        title: Jetty 9.4.33.v20201020 changelog
+    message: |-
+      Winstone 5.12: Update Jetty from 9.4.30.v20200611 to 9.4.33.v20201020.
+      Introduce LowResourceMonitor from Jetty by upgrading to Winstone 5.12.
+  - type: rfe
+    category: rfe
+    pull: 5035
+    issue: 64039
+    authors:
+    - jtnord
+    message: |-
+      Improve performance of authorisation strategies when the authentication realm is case insensitive.
+
+  # pull: 5033 (PR title: JDK 11.0.9 / 8.0.272+ compatible FunctionsTest#printThrowable())
+  # pull: 5064 (PR title: Update the pinned version of ATH in use to a more recent version)
+
+  lts_changes: # compared to lts_predecessor 2.249.3 (selected by personal review)
+
+  - type: major rfe
+    category: major rfe
+    pull: 4909
+    references:
+      - pull: 4909
+      - url: https://github.com/jenkinsci/jep/blob/master/jep/224/README.adoc
+        title: JEP-224
+    authors:
+    - timja
+    message: |-
+      Graduate Overall/SystemRead permission to general availability (GA) status.
+
+  - type: major rfe
+    category: rfe
+    pull: 4951
+    authors:
+    - daniel-beck
+    message: >
+      SSHD module 2.7: Remove deprecated key exchange and MAC algorithms.
```

Comment:
I would recommend adding it to upgrade guidelines, just in case

Reply:
I've made an attempt to add an upgrade guide for the SSHD module in f3b642097604f8fa4cf52c2a9c6ed532d9cd0e07 .  Would love to have a review from you and @daniel-beck to confirm that I've phrased it reasonably.